### PR TITLE
Autistici switched to Let's Encrypt

### PR DESCRIPTION
**Pull request by Devices**
No more self signed certificate.

### CODE REVIEW

**Comment by J0WI** [comment]
Please just use
```
 	<rule from="^http:"
to="https:" />
```
to simplify.

File: src/chrome/content/rules/Autistici-Inventati.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<ruleset name="Autistici/inventati" default_off="self-signed">
+<ruleset name="Autistici/inventati">
 
 	<target host="autistici.org" />
 	<target host="*.autistici.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/Autistici-Inventati.xml]: Wildcrard targets requires at least three test urls.
Use just www otherwise.

**Comment by Devices** [comment]
Like this? I am not sure about your proposed rule change, since I just copied the current rules and only removed the default off thing.

File: src/chrome/content/rules/Autistici-Inventati.xml [hunk 2]
```diff
@@ -12,10 +12,10 @@
...
 										-->
-	<rule from="^http://(?:www\.)?(?:autistic|inventat)i\.org/"
-		to="https://www.autistici.org/" />
+	<rule from="^http:"
+		to="https:" />
 
-	<rule from="^http://([^/:@\.]+)\.(autistici|inventati)\.org/"
-		to="https://$1.$2.org/" />
+	<rule from="^http:"
+		to="https:" />
```
Comment (J0WI) [inline src/chrome/content/rules/Autistici-Inventati.xml]: This block is only needed once.

**Comment by Devices** [comment]
Don't know how I missed that.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**